packages/bundle/src/adaptiveCards/Attachment/AdaptiveCardAttachment.js

```diff
@@ -17,11 +16,9 @@ function stripSubmitAction(card) {
   return { ...card, nextActions };
 }
 
-export default class AdaptiveCardAttachment extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.createAdaptiveCard = memoize((adaptiveCards, content) => {
+const AdaptiveCardAttachment = ({ adaptiveCardHostConfig, adaptiveCards, attachment: { content }, renderMarkdown }) => {
+  const createAdaptiveCard = useMemo(() => {
```

Comment:
I think this is a noun. Something like `cardBuilt`. It should convey the idea that "the card built using JSON and the Adaptive Card renderer".

Return of `useMemo` should reflect the result of the function call. For example, this one returns an object. So this should be noun. If it returns a function, it should be verb.

On the side, `useCallback` always returns a function, so it should be verb.

And please fix others.